tests/spread/dependencies/strict-dependencies-errors/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: test charm build failures with strict dependencies
+
+environment:
+  CHARM/no_requirements: no-requirements/
+  CHARM/extra_packages: extra-packages/
+
+execute: |
+  cd $CHARM
+  # Charmcraft should fail for these charms.
+  ! charmcraft pack --verbose
```

Comment:
is there an error message that you can match against, to protect against this packing failing for reasons _other_ than strict package validation?

Reply:
Any relevant error messages should warn about `charm-strict-dependencies` so I've MATCHed for that